pyproject.toml

```diff
@@ -1,2 +1,2 @@
 [build-system]
-requires = ["setuptools >= 30.4.0", "wheel"]
+requires = ["setuptools >= 38.3.0", "wheel"]
```

Comment:
picked this up while fighting with the sdist. setuptools should be >=38.3 if we want the version with most bugfixes regarding setup.cfg